multi-int-store/src/main/java/com/connexta/multiintstore/adaptors/S3StorageAdaptor.java

```diff
@@ -44,16 +44,16 @@ public S3StorageAdaptor(
 
   @Override
   public void store(
-      @NotBlank final String mimeType,
-      @NotNull final InputStream inputStream,
       @NotNull @Min(1L) @Max(10737418240L) final Long fileSize,
+      @NotBlank final String contentType,
```

Comment:
Content Type generally refers to the HTTP header. I think in this context it makes more sense to use Media Type (formerly MIME Type https://www.iana.org/assignments/media-types/media-types.xhtml)